_episodes/02-software-development.md

```diff
@@ -19,24 +17,137 @@ View (user interface), and Controller (input/output and data manipulation)."
 - "The software project template we use throughout this workshop is an example of an MVC application that manipulates
 patients’ inflammation data and performs basic statistical analysis using Python."
 ---
-## Introduction
+
+## Our Software Project
+So, you have joined a software development team that has been working on the [patient inflammation project](https://github.com/softwaresaved/python-intermediate-inflammation) developed in Python and stored on GitHub. 
+The software project studies inflammation in patients 
+who have been given a new treatment for arthritis and reuses the inflammation dataset from the [novice Software Carpentry Python lesson](https://swcarpentry.github.io/python-novice-inflammation/index.html). The project is not finished and we will be building 
```

Comment:
Feels like there should be a few words about the dataset - if so, only a few words (given learners may not know anything about a prior SWC novice course), and/or what the learner is expected to do with the link - e.g. should they take a quick look to familiarise?

Reply:
Fixed via 6823a37.